components/sysutils/coreutils/Makefile

```diff
@@ -46,21 +52,30 @@ PATCH_LEVEL=		0
 # Prevent gcc register spill error on 32-bit SPARC
 gcc_OPT.sparc.32 =	-O1
 
-CONFIGURE_PREFIX	 =	/usr/gnu
-CONFIGURE_OPTIONS	+=	--libdir=/usr/lib
+# Recreate configure and Makefile to pick up changes in .../src/local.mk
+COMPONENT_PREP_ACTION = (cd $(@D) ; autoreconf -f -i)
+
+CONFIGURE_PREFIX	 =	$(GNUDIR)
+CONFIGURE_BINDIR.64	 =	$(GNUBIN)
+CONFIGURE_OPTIONS	+=	CPPFLAGS=" -I$(USRINCDIR)/gmp"
+CONFIGURE_OPTIONS	+=	--libdir=$(USRLIBDIR.$(BITS))
+# Put libstdbuf in the library directory
+CONFIGURE_OPTIONS	+=	--libexecdir=$(USRLIBDIR.$(BITS))
 CONFIGURE_OPTIONS	+=	--infodir=$(CONFIGURE_INFODIR)
-CONFIGURE_OPTIONS	+=	CPPFLAGS=-I/usr/include/gmp
 
 # Enable ASLR for this component
 ASLR_MODE = $(ASLR_ENABLE)
 
-COMPONENT_POST_INSTALL_ACTION +=	( cp $(COMPONENT_DIR)/files/*.1  \
-	$(PROTO_DIR)/usr/gnu/share/man/man1/ ) 
+COMPONENT_POST_INSTALL_ACTION += ( cp $(COMPONENT_DIR)/files/*.1 \
+                                      $(PROTOGNUSHAREMAN1) ) 
+
+# Build 32-bit for libstdbuf only, then overwrite with 64bit binaries
+build:		$(BUILD_32_and_64)
 
-# common targets
-build:		$(BUILD_32)
+# Make sure 32-bit binaries are installed first. 
+$(INSTALL_64):	$(INSTALL_32)
 
-install:	$(INSTALL_32)
+install:	$(INSTALL_64)
```

Comment:
Hmm I thought you suggested earlier that I would change it, that's why I modified.
I will revert then.